HIP-Basic/CMakeLists.txt

```diff
@@ -25,8 +25,45 @@ project(HIP-Basic LANGUAGES CXX)
 
 # Only supported on HIP (not CUDA)
 if(NOT "${GPU_RUNTIME}" STREQUAL "CUDA")
-    add_subdirectory(assembly_to_executable)
-    add_subdirectory(llvm_ir_to_executable)
+
+    # Make sure the dependencies can be found before building.
+    find_program(
+        LLVM_DIS_COMMAND llvm-dis
+        PATH_SUFFIXES bin
+        PATHS
+        ${ROCM_ROOT}/llvm
+        ${CMAKE_INSTALL_PREFIX}/llvm)
+    find_program(
+        OFFLOAD_BUNDLER_COMMAND clang-offload-bundler
+        PATH_SUFFIXES bin
+        PATHS
+        ${ROCM_ROOT}/llvm
+        ${CMAKE_INSTALL_PREFIX}/llvm)
+    find_program(
+        LLVM_MC_COMMAND llvm-mc
+        PATH_SUFFIXES bin
+        PATHS
+        ${ROCM_ROOT}/llvm
+        ${CMAKE_INSTALL_PREFIX}/llvm)
+
+    if(LLVM_DIS_COMMAND_FOUND AND OFFLOAD_BUNDLER_COMMAND_FOUND AND LLVM_MC_COMMAND_FOUND)
```

Comment:
```suggestion
    if(LLVM_DIS_COMMAND AND OFFLOAD_BUNDLER_COMMAND AND LLVM_MC_COMMAND)
```
find_program() is not creating a "FOUND" variable for these commands. 
In testing, I found that these two examples were being skipped on my system, even though I have "rocm-llvm-dev" installed. Instead the LLVM_DIS_COMMAND variable (for example) will be set to "LLVM_DIS_COMMAND-NOTFOUND" or to the path that "llvm-dis" is found.

Reply:
Ah, I see. Thanks for the explainer. Changed it, and confirmed it worked on my end.